composer.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "localgovdrupal/localgov_guides",
-  "description": "LocalGovDrupal distribution: Guides feature.",
-  "type": "drupal-module",
-  "homepage": "https://github.com/localgovdrupal/localgov_guides",
-  "license": "GPL-2.0-or-later",
-  "minimum-stability": "dev",
-  "require": {
-    "localgovdrupal/localgov_core": "^2.1"
-  },
-  "require-dev": {
-    "localgovdrupal/localgov_services": "^2.0",
-    "localgovdrupal/localgov_topics": "^1.0",
-    "drupal/pathauto": "^1.6"
-  }
+    "name": "localgovdrupal/localgov_guides",
+    "description": "LocalGovDrupal distribution: Guides feature.",
+    "type": "drupal-module",
+    "homepage": "https://github.com/localgovdrupal/localgov_guides",
+    "license": "GPL-2.0-or-later",
+    "minimum-stability": "dev",
+    "require": {
+        "localgovdrupal/localgov_core": "dev-feature/drupal-10 as 2.2.0"
```

Comment:
I'm guessing we could even leave the require-dev being on the services branch, but not the localgov_core